src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -65,6 +65,7 @@ const attributionName = computed(() =>
     layer.value.attributions.map((attribution) => attribution.name).join(', ')
 )
 const showLegendIcon = computed(() => layer.value.hasLegend)
+const hasTimeEntries = computed(() => layer.value.timeConfig?.timeEntries?.length > 0 ?? false)
```

Comment:
You might get away by using `.hasMultipleTimestamps` from `GeoAdminLayer` class, it checks that there is more than 1 timestamp available (sometimes there's only one, that wouldn't make much sense to duplicate such a layer)

Reply:
Good to know, I changed it, thanks